lib/diplomat/session.rb

```diff
@@ -2,47 +2,48 @@
 
 module Diplomat
   class Session < Diplomat::RestClient
-
-    @access_methods = [ :create, :destroy, :list, :renew ]
+    @access_methods = [:create, :destroy, :list, :renew]
 
     # Create a new session
     # @param value [Object] hash or json representation of the session arguments
     # @return [String] The sesssion id
-    def create value=nil
+    def create(value = nil)
       # TODO: only certain keys are recognised in a session create request,
       # should raise an error on others.
       raw = @conn.put do |req|
-        req.url "/v1/session/create"
-        req.body = (if value.kind_of?(String) then value else JSON.generate(value) end) unless value.nil?
+        req.url '/v1/session/create'
+        unless value.nil?
+          value.is_a?(String) ? req.body = value : req.body = JSON.generate(value)
+        end
       end
```

Comment:
This is an ongoing issue - I think we should ignore this issue for now and open a separate PR to address this across the board - problem is that behaviour change should probably be wrapped up into a major release.